### ISSUE TITLE
Tor service is unable to connect to directory server.

### ISSUE THREAD

**Issue by TKone7** [Samourai-Wallet/samourai-dojo#40]
When I run the installation (with my own Bitcoin full node) I see the tor service hung up at 5% with the following error messages:

tor      | Jul 13 01:24:21.000 [warn] Problem bootstrapping. Stuck at 5%: Connecting to directory server. (No route to host; NOROUTE; count 8; recommendation warn; host FE296180018833AF03A8EACD5894A614623D3F76 at 149.56.45.200:9001)
tor      | Jul 13 01:24:21.000 [warn] 7 connections have failed:
tor      | Jul 13 01:24:21.000 [warn]  7 connections died in state connect()ing with SSL state (No SSL object)
tor      | Jul 13 01:24:21.000 [warn] Problem bootstrapping. Stuck at 5%: Connecting to directory server. (No route to host; NOROUTE; count 9; recommendation warn; host 7EA6EAD6FD83083C538F44038BBFA077587DD755 at 194.109.206.212:443)

It retries every minute or so but never succeeds to connect. Someone has any idea what that could be?
Regards
Tobi

**Comment by kenshin-samourai** [comment]
it sounds like a problem with your internet connection.

does your ISP block any access to the internet? (like Tor)

can you browse the web with a Tor browser installed on the machine hosting dojo or on a machine in the same local network?

**Comment by TKone7** [comment]
Hi
When I first researched the problem I came on the same track. But TOR is absolutely no problem from my network (with my ISP). I am able to run TOR on the same machine (outside the container) without any issues. The only difference I can spot are the versions of the packages used:
**Within Docker container:**
Jul 15 09:30:42.377 [notice] Tor 0.3.5.8 (git-5030edfb534245ed) running on Linux with Libevent 2.0.21-stable, OpenSSL 1.1.0k, Zlib 1.2.8, Liblzma N/A, and Libzstd N/A.
**Outside:**
Jul 15 09:39:10.027 [notice] Tor 0.3.2.10 (git-0edaa32732ec8930) running on Linux with Libevent 2.1.8-stable, OpenSSL 1.1.1, Zlib 1.2.11, Liblzma 5.2.2, and Libzstd 1.3.3.
Now that I write this I see that _Liblzma_ and _Libzstd_ print version "N/A" within Docker. Might this be the cause of the problem?
I am still relatively new to Linux, sorry for the basic questions. Your help is much appreciated!
